README.md

```diff
@@ -74,9 +74,8 @@ find what you need, [reach out any time](https://simplyrets.com/#home-contact).
 
 <br/>
 
-## [API examples (GitHub)](https://github.com/SimplyRETS/examples)
+{% include topics.html %}
```

Comment:
This makes it so that the `lastId` post is shown before the `limit`/`offset` post.